pkg/runner/run_context.go

```diff
@@ -612,6 +655,18 @@ func (rc *RunContext) withGithubEnv(env map[string]string) map[string]string {
 	env["GITHUB_SERVER_URL"] = "https://github.com"
 	env["GITHUB_API_URL"] = "https://api.github.com"
 	env["GITHUB_GRAPHQL_URL"] = "https://api.github.com/graphql"
+	env["GITHUB_ACTION_REF"] = github.ActionRef
+	env["GITHUB_ACTION_REPOSITORY"] = github.ActionRepository
+	env["GITHUB_BASE_REF"] = github.BaseRef
+	env["GITHUB_HEAD_REF"] = github.HeadRef
+	env["GITHUB_JOB"] = rc.JobName
+	env["GITHUB_PATH"] = github.Path
+	env["GITHUB_REPOSITORY_OWNER"] = github.RepositoryOwner
+	env["GITHUB_RETENTION_DAYS"] = github.RetentionDays
+	env["RUNNER_PERFLOG"] = github.RunnerPerflog
+	env["RUNNER_TRACKING_ID"] = github.RunnerTrackingID
+	env["RUNNER_USER"] = github.RunnerUser
+	env["USER"] = github.RunnerUser
```

Comment:
I'd rather not overwrite `USER` with `RUNNER_USER`

Reply:
By default, `user` is empty in `act` and expected to be `runner` in gha.

Would you be ok w/ setting it if it's empty as opposed to unconditionally?

Reply:
This is #543

Reply:
Yes but it should be a user that is used in the image, not `runner`

Reply:
I can imagine trying to find the answer by running `whoami` in the container. Is there a better strategy?

Reply:
For the time being, I'm dropping this. Someone can revisit it if it turns out that normal users need something beyond fixed default images.